server/controllers/accounts.js

```diff
@@ -0,0 +1,12 @@
+const models = require('../models/index.js');
```

Comment:
Syntactic note: References to index.js can be omitted since index.js is used by default when we require a folder in node.